simularium/binding-sim-edu
# Feature/events over time plot

Problem
=======
closes #8 

Solution
========
Added a plot that plots the events, I changed it to accumulate the values for every ten time steps so it's a little more readable 


## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)


Steps to Verify:
----------------
1. bun dev
1. click change to simulation view
2. click play
3. should see 2 new plots that are histograms of the events at each ten time points
